requirements.txt

```diff
@@ -17,6 +17,7 @@ pytups==0.86.2
 pytz==2023.3.post1
 requests==2.31.0
 six==1.16.0
+tqdm==4.66.1
```

Comment:
In other projects tqdm output in a log can be ugly because it writes a new line for each percentage. However, I still think is a good idea to have it to make sure process is running. How would you approach the activate/deactivate process? Verbose flag in __ init __ or env variable config or both

Reply:
Lets leave it like this for now but i'm not 100% sold on it